### ISSUE TITLE
Conflict with reaction tracking in Midi-QoL when starting combat with quick combat

### ISSUE THREAD

**Issue by rockwallguy** [veevelder/quick-combat#19]
**Describe the bug**
This is a weird one.  Starting combat with the quick combat hotkey breaks reaction tracking for owned tokens through Midi-QoL.

**To Reproduce**
Start a combat with my hotkey to roll all initiatives (NPCs and PCs).  Cycle through combat until it is an NPC's turn in combat.  Make an attack with an owned token.  There will not be any reaction tracking for the owned token.  If you do the same test for an unowned (NPC) token, reaction tracking will occur normally.  By reaction tracking, I mean the reaction buff applying and getting a warning if you use your reaction twice.  

If, however, you still have quick combat active, but don't use the hotkey to start combat (by manually adding them all to the combat tracker, clicking roll all, then clicking begin combat), reaction tracking works normally.  I've tested this extensively with find the culprit and trying different things to see if I can replicate the problem otherwise.  The only thing I can find that breaks reaction tracking in this way is starting combat through the quick combat hotkey.  I'm using SHIFT-1 as my start hotkey, if that matters.

**Expected behavior**
I'd expect reaction tracking to work normally regardless of how I start combat.

**Screenshots**

**Version Information**
 - Quick Combat: 0.3.6
 - Midi-Qol: 10.0.18
 - FoundryVTT: v10
 - System: Windows 11, self hosting
 - Other Modules: Tested with only Midi-QoL, DAE, libwrapper, socketlib, Dfred's CE, and Quick Combat active.

**Additional context**

**Comment by veevelder** [comment]
strange indeed, I will take a look

**Comment by veevelder** [comment]
I am having trouble recreating your issue, can I get the actor/token relevant (like the reaction and attack stuff) settings of the owned NPC and the owned NPC. Can you provide the Midi-QoL and Dfred's CE settings as well.

If you think it would be easier can you just provide me with a zip of your world and game settings folders i.e. `C:\Users\[username]\AppData\Local\FoundryVTT\Data\worlds\[worldname]` just keep in mind this will include user names and passwords and if you didnt install it in a different location.

**Comment by rockwallguy** [comment]
I'm happy to zip it and share with you.  Or I can just open the server and give you GM access.   I tested it with different actors and in different worlds.  I'm able to reproduce it very consistently.  I can even hop in discord and show you what I'm doing to reproduce it.  

I'm not sure how to send you a private message.  I'm available all day today.  Let me know when you have time and how to contact you privately.

**Comment by veevelder** [comment]
lets just start with the zip first, im not sure how much time ill have other the weekend just yet to work on this. You should be able to just add the zip file to the comment box. if that doesnt work we can figure something else out

**Comment by rockwallguy** [comment]
Upload size was too big for posting here, but here's a google drive link.  

https://drive.google.com/file/d/1tgY3g6WhLlYuiyqhkLgl1ae6JkqDOMPf/view?usp=sharing

**Comment by veevelder** [comment]
Thanks for supplying your world, I did manage to recreate the issue you where seeing with it. And I think I fixed the issue go ahead an update the mod and let me know if it worked or not.

The issue was when going through the game to add the combatants it added extra information that quick-combat did not such as the scene and actor info. I've now added that info and it seems to have fixed the issue.
